packages/component-library/stories/README.md

```diff
@@ -0,0 +1,85 @@
+# CIVIC Design System Story Pattern
+
+Storybook is used to document [CIVIC's Design System](https://hackoregon.github.io/civic/). Each component in the library has a story.
+
+As you read through this document, refer to these stories to see the design pattern in use:
+
+- [Horizontal Bar Chart](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-horizontal-bar-chart--standard)
+- [Line Chart](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-line-chart--standard)
+- [Scatterplot](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-scatterplot--standard)
+- [Scatterplot Map](https://hackoregon.github.io/civic/?path=/story/component-lib-maps-scatterplot-map--standard)
+
```

Comment:
It might be helpful to include the Slider story as well because it's a little different from the others listed.  It uses a wrapper in order to make an onChange event work for shared components.  I'm not familiar enough yet with the variety of components to know if that's likely to be helpful or if most don't need it, in which case it'd be confusing.  What do you think?

Reply:
The StateWrapper helper in the Slider story is for handling state inside a story, which you need to do with controlled form elements (inputs, radios, dropdowns, etc...). Not sure the best place to document that, maybe as a note?

Reply:
Thanks for these comments!  I added a section on Utility functions and included StatefulWrapper as well as getKeyNames.